src/Analyzer.Core.JsonRuleTests/Tests/AppServiceApiApp_HTTPS.json

```diff
@@ -0,0 +1,7 @@
+[
+    {
```

Comment:
I think we should be able to add tests for ExpectedPass = true

Reply:
I did sort of add some examples, but they are implicit.  There are resources defined in the test template that pass, and there are no line numbers reported within those resources.  ExpectPass could be true if a template is added for that purpose.  I didn't do that here.

Reply:
I know the TTK has templates with expected fail scenarios and expected pass scenarios in different folders.